src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/SyncCloudContent.targets

```diff
@@ -0,0 +1,31 @@
+<Project ToolsVersion="12.0" DefaultTargets="DownloadBlobsFromAzureTargets" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask TaskName="DownloadFromAzure" AssemblyFile="$(ToolsDir)net45/Microsoft.DotNet.Build.CloudTestTasks.dll"/>
+  <UsingTask TaskName="ListAzureContainers" AssemblyFile="$(ToolsDir)/net45/Microsoft.DotNet.Build.CloudTestTasks.dll"/>
+
+  <PropertyGroup>
+    <DownloadDirectory Condition="'$(DownloadDirectory)' == ''">$(PackagesDir)/AzureTransfer</DownloadDirectory>
+  </PropertyGroup>
+
+  <Target Name="DownloadBlobsFromAzureTargets" DependsOnTargets="GetLatestContainerNameFromAzure;DownloadBlobsFromAzure" />
+
+  <!-- Grab latest azure container if a specific azure container is not specified -->
+  <Target Name="GetLatestContainerNameFromAzure" Condition="'$(ContainerName)' == ''">
+    <ListAzureContainers AccountName="$(CloudDropAccountName)"
+                         AccountKey="$(CloudDropAccessToken)"
+                         Prefix="$(ContainerNamePrefix)">
+       <Output TaskParameter="ContainerNames" ItemName="AzureContainerName" />
+    </ListAzureContainers>
+    <Message Importance="Normal" Text="Container names: %(AzureContainerName.Identity)" />
```

Comment:
Do we need this message? If so, should we set the importance to low so that it only shows on verbose logs?